samples/image-metadata/src/components/FocalPointDialog/FocalPointDialog.js

```diff
@@ -0,0 +1,157 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Subheading, Modal } from '@contentful/forma-36-react-components';
+
+import { FocalPoint } from '../FocalPoint';
+import { ImagePreviewWithFocalPoint } from '../ImagePreviewWithFocalPoint';
+
+import { MAX_PREVIEW_WRAPPER_SIZE, styles } from './styles';
+
+export class FocalPointDialog extends Component {
+  static propTypes = {
+    file: PropTypes.object.isRequired,
+    focalPoint: PropTypes.shape({
+      x: PropTypes.number.isRequired,
+      y: PropTypes.number.isRequired
+    }),
+    onClose: PropTypes.func.isRequired,
+    onSave: PropTypes.func.isRequired,
+    sdk: PropTypes.object.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+    this.imgRef = React.createRef();
+    this.onWindowResize = () =>
```

Comment:
I see that you use class properties everywhere but `imgRef` and `onWindowResize`. Let's make it consistent and move these things out of constructor as well.

Reply:
Done